src/Core/Settings.h

```diff
@@ -488,6 +488,7 @@ class IColumn;
     M(Bool, enable_job_stack_trace, false, "Output stack trace of a job creator when job results in exception", 0) \
     M(Bool, allow_ddl, true, "If it is set to true, then a user is allowed to executed DDL queries.", 0) \
     M(Bool, parallel_view_processing, false, "Enables pushing to attached views concurrently instead of sequentially.", 0) \
+    M(Bool, parallel_view_query_execution, false, "Enables asynchronous view query execution on pushing to attached views.", 0) \
```

Comment:
I don't like `asynchronous` wording here, maybe parallel will be better?

```suggestion
    M(Bool, parallel_view_query_execution, false, "Enables parallel view query execution on pushing to attached views.", 0) \
```